test/active_model_serializers/adapter_for_test.rb

```diff
@@ -1,5 +1,7 @@
+require 'test_helper'
+
 module ActiveModelSerializers
-  class AdapterForTest < ActiveSupport::TestCase
```

Comment:
See commit.  Doesn't need to be in this PR, but doesn't really deserved its own.  Just something I saw when running files individually